### ISSUE TITLE
refactor imports

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#337]
some import lines (especially from helpers import ...) tend to get rather long.

could be globally refactored to be like:
```
import helpers as h
...
h.foo()
```

**Comment by RonnyPfannschmidt** [comment]
one might avoid that in hot loops tho, due to getattr costs

**Comment by ThomasWaldmann** [comment]
Implement in 1.0?

**Comment by ThomasWaldmann** [comment]
also, that from helpers import is a constant PITA when merging  branches due to conflicts.

**Comment by hansmi** [comment]
In a previous project of mine we had the policy of one import per line, i.e.

```
from .helpers import foo
from .helpers import bar
```

This made resolving conflicts easier.

**Comment by ThomasWaldmann** [comment]
We'ld rather just "import .helpers".

**Comment by ThomasWaldmann** [comment]
implemented by #1112.

@hansmi I decided to take a mixed approach: not use h.foo (ugliness all over the code), but also not 1 import per line approach, but group a few related imports into 1 line.
